api/v1alpha1/etcd_types.go

```diff
@@ -43,6 +55,10 @@ type GarbageCollectionPolicy string
 // StorageProvider defines the type of object store provider for storing backups.
 type StorageProvider string
 
+// Policy defines the type of compressionPolicy for compression of snapshots.
+// +kubebuilder:validation:Enum=gzip;lzw;zlib
+type Policy string
```

Comment:
s/Policy/CompressionPolicy, since `Policy` is too generic a name to use. Also change all relevant occurrences.